src/app/components/Overview/Overview.tsx

```diff
@@ -1,43 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { observer } from 'mobx-react';
 import Grid from '@material-ui/core/Grid';
+import { useHistory } from 'react-router-dom';
+import config from '~app/common/config';
 import Layout from '~app/common/components/Layout';
-import { useStyles } from '~app/components/Styles';
-// import Banner from '~app/common/components/Banner';
+import { Button } from '~app/common/components/Button';
 import Hero from '~app/components/Overview/components/Hero';
 import Stats from '~app/components/Overview/components/Stats';
-import Paper from '~app/components/Overview/components/Paper';
 import Column from '~app/components/Overview/components/Column';
 import Header from '~app/components/Overview/components/Header';
+import { useStyles } from '~app/components/Overview/Overview.styles';
 import Container from '~app/components/Overview/components/Container';
+import HeaderWrapper from '~app/components/Overview/components/HeaderWrapper';
 import { OperatorsTable, ValidatorsTable } from '~app/components/Overview/components/Tables';
 
 const Overview = () => {
+  const history = useHistory();
   const classes = useStyles();
-  const tableContainerStyle = {
-    border: '1px solid #5B6C84',
+  const [operatorsExist, setOperatorsExist] = useState(false);
+  const [validatorsExist, setValidatorsExist] = useState(false);
+  
+  const goTo = (url: string) => {
+    history.push(url);
```

Comment:
Why to use this wrapper if there is no extra logic, you can use directly `history.push`

Reply:
done